tests/airgapped/training/README.md

```diff
@@ -0,0 +1,31 @@
+# Testing training operator in airgapped
+
+This directory is dedicated to testing training operator in an airgapped environment.
+
+## Prerequisites
+
+Prepare the airgapped environment and deploy CKF by following the steps in [Airgapped test scripts](../README.md#testing-airgapped-installation).
```

Comment:
The image for `kfp-profile-controller` was outdated in the `./scripts/airgapped/deploy-1.8.sh` script, so I had to update it for the charm to deploy properly.
Also, the second part in the `Configure the dashboard` section of the README.md can be removed, since the `juju config` commands are now run as part of the script.

Reply:
I have addressed these comments in another PR #964 since this PR is about the training operator test